CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Changed
 - [WIP] `torch` is loaded lazily
+- `SearchSpace`s now use `ndarray` instead of `Tensor` 
```

Comment:
Just a quick note: I'll review your PRs tomorrow morning, leaving this here as a blocker comment